<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreGamePrompt</class>
 <widget class="QDialog" name="PreGamePrompt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>217</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plane Setup</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,1,0">
   <item row="0" column="0">
    <widget class="QGroupBox" name="plane_settings_grp">
     <property name="title">
      <string>Plane Settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="2" column="1">
       <widget class="QSpinBox" name="starting_speed">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="has_bombs">
        <property name="text">
         <string>Is carrying a bomb load?</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Starting Altitude:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" rowspan="2">
       <widget class="QSpinBox" name="starting_alt"/>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="bomb_count">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="suffix">
         <string> bombs</string>
        </property>
        <property name="value">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Starting Speed:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="crew_names_grid">
     <property name="title">
      <string>Crew</string>
     </property>
     <layout class="QGridLayout" name="gridLayout"/>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>PreGamePrompt</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>137</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>has_bombs</sender>
   <signal>toggled(bool)</signal>
   <receiver>bomb_count</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
